internal/sdkprovider/userconfig/converters/converters.go

```diff
@@ -23,7 +23,10 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-const userConfigSuffix = "_user_config"
+const (
+	userConfigSuffix   = "_user_config"
+	AllowIPFilterPurge = "AIVEN_ALLOW_IP_FILTER_PURGE"
```

Comment:
Why do we make it an exported const? I don't know if it's a good idea if we only use it in the test, maybe the test should also define the const? WDYT?

Reply:
`AllowIPFilterPurge` is used to format the error message too.

Reply:
Yes, I can see, but I mean why is it an exported const? We can make it a non-exported const instead, and introduce a separate one in the test, WDYT?

Reply:
The idea of constants is to be the source of truth.